BLSQ/iaso
# IA-2131  / IA-2133: API to return orgunit without submissions

fix IA-2131 API to return orgunit without submissions
fix also IA-2133

this is needed for the Registry page "Missing feature submissions"
After discussions we are implementing that in the completeness api

This add a new argument `without_submissions` that only return the orgunit who have no submission for the queryied `form_id`(s)

Example call /api/v2/completeness_stats/?page=1&limit=50&form_id=6&parent_org_unit_id=730&without_submissions=true
this is paginated and accept the same habitual arguments for the completess_api (See swagger  for doc )

Note that per default the completeness api only return the direct
children, pass the `org_unit_type_ids` if you want all the descendants
of that type(s)



## Self proofreading checklist

- [NA] New translations have been added or updated if new strings have been introduced in the frontend
- [NA] My migrations file are included

## Changes
see above
## How to test

Find an orgunit whose child should enter a submission but hasn't  (red cross) in the completeness ui, check what network call it's making and add the arguments. or let Christophe implement the front



Adding test later
